tools/build_defs/repo/git.bzl

```diff
@@ -165,34 +162,29 @@ _new_git_repository_attrs = dict(_common_attrs.items() + {
             "Either `workspace_file` or `workspace_file_content` can be " +
             "specified, or neither, but not both.",
     ),
-}.items())
-
-def _new_git_repository_implementation(ctx):
```

Comment:
We should just get rid of this so there is no documenetntation about new_git_repository.
An assignment at the end can do it.  

I'll make that happen in the merge.